index.html

```diff
@@ -56,9 +49,9 @@ <h2 class="aboutus__title">
         </div>
         <div class="aboutus__main">
             <div class="aboutus__main-imgs">
-                <img class="aboutus__photo" src="" alt="">
+                <img class="aboutus__photo" src=".//aboutus__img/55379f057320e498706fd94c70f4cb97.png" alt="">
                 <div class="aboutus__main-mid">
-                    <img class="aboutus__photo--ratio-1x1" src="" alt="">
+                    <img class="aboutus__photo--ratio-1x1" src=".//aboutus__img/b36ed8d7e8280cf5d455e3bafcbdb460.png" alt="">
```

Comment:
jak dajesz modyfikator do klasy to główna klasa powinna być jako 1 
`aboutus__photo aboutus__photo--ratio-1x1`

Reply:
dodaj też alty do wszystkich zdjęć